### PR TITLE
Warning note when not using label for a searchbox

### PR DESCRIPTION
**Pull request by notabene**
(also my editor got rid of some loose empty spaces at the end of lines – I hope it's not too inconvenient)

***
<a href="https://pr-preview.s3.amazonaws.com/notabene/aria/pull/1170.html#aria-placeholder" title="Last updated on Feb 7, 2020, 6:09 AM UTC (176fd8a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/aria/1170/6ec58d5...notabene:176fd8a.html" title="Last updated on Feb 7, 2020, 6:09 AM UTC (176fd8a)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11909,6 +11909,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
...
 				<p class="note">As is the case with the related <code>placeholder</code> attribute in [[HTML]], use of placeholder text as a replacement for a displayed label can reduce the accessibility and usability of the control for a range of users including older users and users with cognitive, mobility, fine motor skill or vision impairments. While the hint given by the control's label is shown at all times, the short hint given in the placeholder attribute is only shown before the user enters a value. Furthermore, placeholder text may be mistaken for a pre-filled value, and as commonly implemented the default color of the placeholder text provides insufficient contrast and the lack of a separate visible label reduces the size of the hit region available for setting focus on the control.</p>
+				<p class="note">The following examples could use the basic <code>&lt;label for="birthday"&gt;</code>/<code>&lt;input id="birthday"&gt;</code> construction, but here the <code>contenteditable</code> area does not allow for <code>label</code>. Association is thus achieved via <code>aria-labelledby</code>.</p>
```
Comment (jnurthen) [inline index.html]: Adding an extra piece of example code seems confusing. Please simplify. Perhaps - "the following examples do not use the HTML label element as it cannot be used to label HTML elements with contenteditable."

**Comment by notabene** [comment]
> I can't consider merging this unless the whitespace changes get cleaned up

Sorry, I'm not completely fluent in English.

Does this mean that you'd want index.html to go back to where it was with all the trailing spaces, or that you wouldn't like to have trailing spaces and the corrections my editor madeare OK?

**Comment by jnurthen** [comment]
> > I can't consider merging this unless the whitespace changes get cleaned up
> 
> Sorry, I'm not completely fluent in English.
> 
> Does this mean that you'd want index.html to go back to where it was with all the trailing spaces, or that you wouldn't like to have trailing spaces and the corrections my editor madeare OK?

Sorry for the confusion.

The PR must only change the lines you mean to change. All other lines must be the same as they were before.

**Comment by notabene** [comment]
> > > I can't consider merging this unless the whitespace changes get cleaned up
> > 
> > 
> > Sorry, I'm not completely fluent in English.
> > Does this mean that you'd want index.html to go back to where it was with all the trailing spaces, or that you wouldn't like to have trailing spaces and the corrections my editor madeare OK?
> 
> Sorry for the confusion.
> 
> The PR must only change the lines you mean to change. All other lines must be the same as they were before.

OK, since I don't know how to do that can you close/reject this PR and I'll try to do another, cleaner one?
(Sorry for the inconvenience, I don't code as much as one would hope.)

**Comment by notabene** [comment]
Better, I hope. Thanks for your suggestion, it's easier to read now.

File: index.html [hunk 2]
```diff
@@ -11909,6 +11909,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
...
 				<p class="note">As is the case with the related <code>placeholder</code> attribute in [[HTML]], use of placeholder text as a replacement for a displayed label can reduce the accessibility and usability of the control for a range of users including older users and users with cognitive, mobility, fine motor skill or vision impairments. While the hint given by the control's label is shown at all times, the short hint given in the placeholder attribute is only shown before the user enters a value. Furthermore, placeholder text may be mistaken for a pre-filled value, and as commonly implemented the default color of the placeholder text provides insufficient contrast and the lack of a separate visible label reduces the size of the hit region available for setting focus on the control.</p>
+				<p class="note">The following examples do not use hte HTML `label` element as it cannot be used to label HTML elements with `contenteditable`.</p>
```
Comment (carmacleod) [inline index.html]:
```suggestion
				<p class="note">The following examples do not use the HTML `label` element as it cannot be used to label HTML elements with `contenteditable`.</p>
```

**Comment by jnurthen** [comment]
Marked as non substantive for IPR from ash-nazg.

File: index.html [hunk 3]
```diff
@@ -11909,6 +11909,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
...
 				<p class="note">As is the case with the related <code>placeholder</code> attribute in [[HTML]], use of placeholder text as a replacement for a displayed label can reduce the accessibility and usability of the control for a range of users including older users and users with cognitive, mobility, fine motor skill or vision impairments. While the hint given by the control's label is shown at all times, the short hint given in the placeholder attribute is only shown before the user enters a value. Furthermore, placeholder text may be mistaken for a pre-filled value, and as commonly implemented the default color of the placeholder text provides insufficient contrast and the lack of a separate visible label reduces the size of the hit region available for setting focus on the control.</p>
+				<p class="note">The following examples do not use the HTML `label` element as it cannot be used to label HTML elements with `contenteditable`.</p>
```
Comment (mcking65) [inline index.html]: Is respec good with the ` characters? Shouldn't this be:
```suggestion
				<p class="note">The following examples do not use the HTML <code>label</code> element as it cannot be used to label HTML elements with <code>contenteditable</code>.</p>
```

Comment (scottaohara) [reply index.html]: respec does convert the ` characters to `<code>...</code>`

but the ARIA spec doesn't use them anywhere else.   probably a good idea to change to be consistent.

**Pull request merged**